qiskit/ignis/experiments/quantum_volume/qv_experiment.py

```diff
@@ -0,0 +1,204 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+Quantum volume Experiment.
+"""
+
+import copy
+import itertools
+import warnings
+
+import numpy as np
+import uuid
+import qiskit
+
+from qiskit.circuit.library import QuantumVolume
+from qiskit.circuit.quantumcircuit import QuantumCircuit
+
+from typing import Optional, Dict, Union, List
+from qiskit.ignis.experiments.base import Experiment, Generator, Analysis
+
+from qiskit import transpile, assemble
+from qiskit.providers import BaseJob
+from qiskit.exceptions import QiskitError
+from qiskit.providers import BaseBackend
+
+
+class QuantumVolumeExperiment(Experiment):
+    """Quantum volume experiment."""
+
+    # pylint: disable=arguments-differ
+    def __init__(self,
+                 qubits: Union[int, List[int]],
+                 job: Optional = None,
+                 seed: Optional(int)=None):
+
+        if isinstance(qubits, int):
+            qubits = [range(qubits)]
+        self._qubits = qubits
+        self._trials = 0
+        self._seed = seed
+        self._circuits = [[] for _ in range(len(self._qubits))]
+        self._circuit_no_meas = [[] for _ in range(len(self._qubits))]
```

Comment:
I am not sure if these should be different array of circuits, or added to "circuits" array, with different metadat

Reply:
Perhaps it makes sense to make them in different arrays, since they are executed on different backends.